pkgs/applications/networking/cluster/k3s/builder.nix

```diff
@@ -184,6 +185,16 @@ let
     src = k3sRepo;
     vendorHash = k3sVendorHash;
 
+    patches =
+      # Disable: Add runtime checking of golang version
+      lib.optional (lib.versionAtLeast k3sVersion "1.28")
+        (fetchpatch {
+          # https://github.com/k3s-io/k3s/pull/9054
+          url = "https://github.com/k3s-io/k3s/commit/b297996b9252b02e56e9425f55f6becbf6bb7832.patch";
+          hash = "sha256-xBOY2jnLhT9dtVKtq26V9QUnuX1q6E/9UcO9IaU719U=";
+          revert = true;
+        });
```

Comment:
```suggestion
      lib.optionals (lib.versionAtLeast k3sVersion "1.28") [
        (fetchpatch {
          # https://github.com/k3s-io/k3s/pull/9054
          url = "https://github.com/k3s-io/k3s/commit/b297996b9252b02e56e9425f55f6becbf6bb7832.patch";
          hash = "sha256-xBOY2jnLhT9dtVKtq26V9QUnuX1q6E/9UcO9IaU719U=";
          revert = true;
        })
      ];
```

Reply:
I don't know what to answer you. Someone could just say the opposite if I had used `lib.optionals` as it is a single item and the extra `[ ]` being unnecessary. Which has happened before (no kidding).
There is no extra patch to be included there. Usually I wouldn't mind doing what is asked. But both commits depend on each other and the hash of commit changing would require changes at other places that cherry-pick these commits. [This PR is long overdue and is causing issues in other places.] As it has no functional effect but to create new problems. I'd rather leave this as it is. I hope you understand.

Reply:
No problem, thanks for the explanation.